Program.cs

```diff
@@ -7,7 +7,7 @@ class Program
         static void Main(string[] args)
         {
             Random r = new Random();
-            int n = r.Next(100);
+            int n = r.Next(1, 100);
```

Comment:
no, w sumie to ma sens

Reply:
https://i.kym-cdn.com/entries/icons/original/000/023/397/C-658VsXoAo3ovC.jpg
:D